### ISSUE TITLE
GPL 3 migration

### ISSUE THREAD

**Issue by anuradhawick** [ljdursi/poapy#12]
@ljdursi I would like to borrow some of your code for a similar task. I plan to use GPLv3.

Did you pick GPL v2 for a specific requirement?
Do you have plans to upgrade to GPLv3?

**Comment by ljdursi** [comment]
Hi - glad some of this code might be of some use to you!  No, I didn't have any particular reason for GPL v2, happy to relicense under v3 - I'm going to mark this done with commit #[8d399c5](https://github.com/ljdursi/poapy/commit/8d399c51e1469e41f63e4fabc798278076cf08b4) but feel free to share how you used some of this code!

**Comment by anuradhawick** [comment]
I am generalising the algorithm for generic applications, including minimiser space POA. I could not find a library that would let me align arbitrary objects, so I made one.

Your blog and code base was very useful. I have acknowledged and followed the license guide but let me know if anything needs to be changed.

[https://github.com/anuradhawick/generic-poa](https://github.com/anuradhawick/generic-poa)

Later, I plan to do some optimisations like banded alignment, but for now it should do what I wanted to do.

Thanks!
